src/mlpack/methods/reinforcement_learning/environment/acrobot.hpp

```diff
@@ -85,19 +85,27 @@ class Acrobot
   /*
    * Implementation of action for Acrobot
    */
-  enum Action
+  class Action
   {
-    negativeTorque,
-    zeroTorque,
-    positiveTorque,
+   public:
+    enum _action
```

Comment:
Do you mind to change the name here, that is in accordance with the mlpack style; maybe `actions` works as well?

Reply:
Yeah it works, I'll use that. Thanks :)